tests/lib/cli_util_test.py

```diff
@@ -59,6 +59,10 @@ def test_parse_key_target(self):
         for spec, expected_parse in cases:
             self.assertEqual(cli_util.parse_key_target(spec), expected_parse)
 
+        usage_error_cases = [':', 'a:']
```

Comment:
This seems different enough from the test cases before -- can you make this into a different function? For example, call it `test_parse_key_target_invalid`

Reply:
Yes, I originally also thought about this. But it seems that all the tests in this file try to group their test cases (the succeeded ones + failed ones) together in this way. I just want to keep them consistent.

Reply:
No problem (I personally think that grouping succeeding + failed test cases together is a bad practice though :wink: )